spec/features/work_packages/details/relations/primerized_relations_spec.rb

```diff
@@ -148,7 +148,7 @@ def label_for_relation_type(relation_type)
 
       tabs.expect_counter("relations", 6)
 
-      relations_tab.expect_relation(relation_follows)
+      relations_tab.expect_closest_relation(relation_follows)
```

Comment:
This isn't necessarily the right place to test for this - and this expectation isn't particularly robust, given that there is only one "follows" relation in this scenario.

Reply:
> When you say that it's not robust, I'm not sure to get it. In which ways could this assertion fail?

"Robust" might have been the wrong choice of words - a bit of an exaggeration. Rather my point was that there is a single "follows" relation in this scenario, so we are not yet testing displaying multiple predecessors at a feature level (and any applicable logic regarding labelling, sorting - tbd - and grouping).

Reply:
Ok got it. Well, I think that's cheap so it's fine.